src/ops/ops.go

```diff
@@ -195,6 +196,27 @@ func (o *ops) Reboot() error {
 	return nil
 }
 
+func (o *ops) SetBootOrder(device string) error {
```

Comment:
Please don't return an error from this function yet.
I think we should start with "best effort" mode so in case we fail to set the boot order the installation won't fail.
This way it can merge today, and we can see it in action in prod.
Once we gain some certainty about what happens at customers sites, we can add the return err

Reply:
Done